util/chplenv/chpl_jemalloc.py

```diff
@@ -1,26 +1,60 @@
 #!/usr/bin/env python3
 import os
 import sys
+import optparse
 
-import chpl_mem, overrides, third_party_utils
+import chpl_mem, chpl_platform, overrides, third_party_utils
 from utils import error, memoize, run_command, warning
 
 
 @memoize
-def get():
-    jemalloc_val = overrides.get('CHPL_JEMALLOC')
-    mem_val = chpl_mem.get('target')
-    if not jemalloc_val:
-        if mem_val == 'jemalloc':
+def get(flag='target'):
+    chpl_jemalloc = overrides.get('CHPL_JEMALLOC')
+    chpl_host_jemalloc = overrides.get('CHPL_HOST_JEMALLOC')
+    chpl_target_jemalloc = overrides.get('CHPL_TARGET_JEMALLOC')
+    mem_val = chpl_mem.get(flag)
+
+    platform_val = chpl_platform.get('target')
```

Comment:
Should platform_val be based on `flag` instead of hardcoding to `target `?

Reply:
yes good catch